build_docs.py

```diff
@@ -1043,7 +993,7 @@ def proofread_canonicals(www_root: Path, skip_cache_invalidation: bool) -> None:
             if not skip_cache_invalidation:
                 url = str(file).replace("/srv/", "https://")
                 logging.info("Purging %s from CDN", url)
-                requests.request("PURGE", url)
+                requests.request("PURGE", url, timeout=30)
```

Comment:
Potentially add a purge helper? Especially given we now want to have consistency over the timeout arg etc.

Reply:
Nice idea, it gets way more readable than the curl-brace-expansion-inside-fstring.

Reply:
done.